tpke/benches/benchmarks.rs

```diff
@@ -15,6 +15,7 @@ pub fn bench_decryption(c: &mut Criterion) {
         msg_size: usize,
     ) -> impl Fn() {
         let rng = &mut rand::rngs::StdRng::seed_from_u64(0);
+        let aad: &[u8] = "my-aad".as_bytes();
```

Comment:
Does the size of the AAD have an impact on the size of the resulting ciphertext?   It might be useful to show that in your benchmarks, or at least use a test AAD content similar to the size that will be used in an implementation of CBD.

Reply:
AAD size shouldn't matter, in the end it's just hashed.